src/components/DigitalConverterComponent.cpp

```diff
@@ -0,0 +1,115 @@
+#include <iostream>
+
+#include <hyro/DigitalConverterComponent.h> 
+
+using namespace std::chrono_literals;
+
+namespace hyro
+{
+std::shared_ptr<HyroLogger> DigitalConverterComponent::s_logger = HyroLoggerManager::CreateLogger("DigitalConverterComponent");
+
+Result
+DigitalConverterComponent::init (const ComponentConfiguration & configuration)
+{
+
+  m_thresholding = Thresholding();
```

Comment:
There is no need to assign a new instance of `Thresholding`  to `m_thresholding`.
You can remove it from here.